docs/library/utime.rst

```diff
@@ -227,3 +227,13 @@ Functions
       hardware also lacks battery-powered RTC, so returns number of seconds
       since last power-up or from other relative, hardware-specific point
       (e.g. reset).
+
+
+.. function:: adjtime([(secs, usecs)])
+
+   This function is only available in the esp32 port. It uses the API's
```

Comment:
s/API/ESP-IDF/ ?

Reply:
mind fixing this? there are many APIs around whereas saying "the ESP-IDF's function" is unambiguous...